src/fields/BooleanField.cpp

```diff
@@ -0,0 +1,15 @@
+#include "BooleanField.h"
+

```

Comment:
For consistency, should `using namespace std;` be included in this file? 
Or removed from files: Field.h, BooleanField.h, etc.